### PR TITLE
235 dedicated page for model cards

### PR DESCRIPTION
**Pull request by vijulshah**

### CODE REVIEW

File: src/ml_board/frontend/Dashboard/src/components/modelCard/ModelCard.tsx [hunk 1]
```diff
@@ -0,0 +1,431 @@
...
+    //             pdf.save(`Experiment_${experiment_id}_ModelCard.pdf`);
+    //             setTableRows(0);
+    //         })
+    //         .catch((error) => {
+    //             console.error('Error saving PDF:', error);
+    //             setTableRows(0);
+    //             // TODO: Show error snackbar
+    //         });
+    //     }, 200);
+    // }
```
Comment (le1nux) [inline src/ml_board/frontend/Dashboard/src/components/modelCard/ModelCard.tsx:158]: legacy code

File: src/ml_board/frontend/Dashboard/src/components/modelCard/ModelCard.tsx [hunk 2]
```diff
@@ -0,0 +1,431 @@
...
+            width: 100%;
+            height: 100%;
+            object-fit: cover;
+        `;
+
+        const card_feel = `
+            box-shadow: 0px 0px 3px 2px rgb(0, 0, 0, 0.2);
+            border-radius: 5px;
+            margin-bottom: 20px;
+        `;
```
Comment (le1nux) [inline src/ml_board/frontend/Dashboard/src/components/modelCard/ModelCard.tsx:196]: can we move this to a css file?

**Review by le1nux — APPROVED** [review]
Nice code. Well separated :-)

**Comment by vijulshah** [comment]
@le1nux, @OsamaMSoliman  I have resolved Merge Conflicts & Changed the Download btn to a floating btn for the model card (as discussed in the meeting). Pls have a look and merge the changes.

**Review by le1nux — APPROVED** [review]
lgtm :-)

**Pull request merged**